README.md

```diff
@@ -123,6 +123,18 @@ for changes to scripts or stylesheets.
 When adding a new library or client-side dependency through npm don't forget to require the scripts in `application.js`
 and the stylesheet in `vendor.scss`.
 
+#### Websockets
+
+In some parts of the website we use websockets to update the page in real-time.
+In development, we use the [Reverb](https://reverb.laravel.com/) websocket server.
```

Comment:
Why do we not use Reverb for production as well? Using two different websocket servers seems convoluted.

Reply:
Reverb does not play nice with our webhosting. Since the two communicate via the laravel adapter there are no differences between using the two. For local it is way easier using reverb, and for prod we should not be going over the ably free tier.